backend/vendor/assets/javascripts/jquery.jconfirm-1.0.min.js

```diff
@@ -0,0 +1,33 @@
+ * You should have received a copy of the GNU
+ * General Public License along with this
+ * program.  If not, see
+ * <http://www.gnu.org/licenses/gpl-3.0.txt>.
+ *
+ * @author Simon Emms <simon@simonemms.com>
+ * @licence GNU GPLv3 <http://www.gnu.org/licenses/gpl-3.0.txt>
+ * @version 1.0
+ */
+(function(e){var t={getConfig:{cancel:"%cancel%",confirm:"%confirm%",cancelTrigger:"%cancelTrigger%",confirmTrigger:"%confirmTrigger%",message:"%message%",title:"%title%",wrapperId:"%wrapperId%",outerWrapperId:"%outerWrapperId%"},getDefaultTemplate:function(){var e=t.getConfig;var n='<div id="'+e.outerWrapperId+'" class="jconfirm_wrapper">';n+='<div id="'+e.wrapperId+'" class="jconfirm_inner">';n+='<a href="#" class="jconfirm_close">×</a>';n+='<h4 class="jconfirm_title">';n+=e.title;n+="</h4>";n+='<div class="jconfirm_message">';n+=e.message;n+="</div>";n+='<div class="jconfirm_buttons">';n+='<a href="#" id="'+e.confirmTrigger+'" class="jconfirm_success">';n+=e.confirm;n+="</a>";n+='<a href="#" id="'+e.cancelTrigger+'" class="jconfirm_fail">';n+=e.cancel;n+="</a>";n+="</div>";n+="</div>";n+="</div>";return n},init:function(n,r){var i=e(window).width()/2;if(i>n.maxWidth){i=n.maxWidth}var s=i/2;if(n.template===null){n.template=t.getDefaultTemplate()}var o=(new Date).getTime();n.cancelTrigger=n.cancelTrigger+"_"+o;n.confirmTrigger=n.confirmTrigger+"_"+o;n.wrapperId=n.wrapperId+"_"+o;var u=t.parseTemplate(n,o);e("body").append(u);var a=e("#"+n.wrapperId).height();var f=a;e("#"+n.wrapperId).css({position:"absolute",top:"50%",left:"50%","margin-left":0-s,"margin-top":0-f/2,width:i,height:a});e("body").keydown(function(t){if(t.keyCode===27){e("#"+n.cancelTrigger).trigger("click");return false}else if(t.keyCode===13){e("#"+n.confirmTrigger).trigger("click");return false}else{return true}});e("#"+n.cancelTrigger).click(function(){e("#"+n.outerWrapperId).remove();return false});e("#"+n.wrapperId).find(".jconfirm_close").click(function(){e("#"+n.cancelTrigger).trigger("click");return false});if(n.closeOnOuterWrapperClick){e("#"+n.outerWrapperId).click(function(){e("#"+n.outerWrapperId).remove();return false});e("#"+n.wrapperId).click(function(){return false})}e("#"+n.confirmTrigger).click(function(){e("#"+n.cancelTrigger).trigger("click");return r()});return false},parseTemplate:function(n,r){var i=n.template;var s=t.getConfig;var o;e.each(s,function(e,t){o=n[e];if(o===null||o===undefined){o=""}i=i.replace(t,o)});return i}};e.jconfirm=function(n,r){var i=e.extend({cancel:"Cancel",cancelTrigger:"jconfirmCancelTrigger",confirm:"OK",confirmTrigger:"jconfirmConfirmTrigger",closeOnOuterWrapperClick:false,maxWidth:500,message:null,outerWrapperId:"jconfirmOuter",template:null,title:"Are you sure?",wrapperId:"jconfirmWrapper"},n);if(typeof n==="function"){r=n}else if(n==="getConfig"){return t.getConfig}else if((typeof n==="object"&&typeof r==="function")===false){e.error("jConfirm incorrectly initialized.  Please check the documentation")}return t.init(i,r)}})(jQuery)
```

Comment:
Line is too long.<br>Mixed double and single quotes.<br>Missing semicolon.<br>Missing '()' invoking a constructor.<br>'r' is defined but never used.